### DISCUSSION TITLE
关于抽卡记录中四星/A级奖励

### DISCUSSION THREAD

**Discussion by BigSillyTiger** [lgou2w/HoYo.Gacha#46] [General]
感谢制作者的工作！
想询问一下，未来更新中，可否有计划四星/A级的奖励也罗列出来呢？
呈现方式比如，某Up池子：SAAAASAA，这种方式

**Comment by lgou2w** [comment]
会有这方面的功能，详见：https://github.com/lgou2w/HoYo.Gacha/issues/28
